### PR TITLE
Church and COVID-19

### PR DESCRIPTION
**Pull request by genebean**

### CODE REVIEW

File: _drafts/church-and-covid-19.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+### The Oasis
+
+Ending up with a mobile setup had a fringe benefit: it opened up options for where we streamed from because once portable there really isn't any difference between the sanctuary and somewhere else. Our revised plan ended up being to stream from an area called The Oasis that is a sitting area in one of our buildings. This actually worked pretty well. The seating is setup in a horse shoe configuration. Our pastor sat on one side on a couch between a couple of windows and was flanked by our youth minister and our children's minister, each in a chair near the end of the couch. Our associate minister of pastoral care was in a chair next to the children's minister on the short part of the horse shoe. To his right on another couch was a couple that serves as our music minister and director of Christian education. In the middle of the sitting area is a round coffee table that we were able to place the Lenten wreath. On the open end of the horse shoe is a long countertop with bookcases to either side of a TV. After some trail and error we decided to frame the children's minister, pastor, and youth minister in the shot and let the others be off-camera.
+
```
Comment (jcoconnor) [inline _drafts/church-and-covid-19.md]: Minor typo here - you have ```trail and error``` should be ```trial and error```

File: _drafts/church-and-covid-19.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+### Synchronization
+
+The rest of the rehearsal went pretty well with the exception of one glaring technical issue: the audio and video were noticeably out-of-sync on the test stream... as in a second or two out from each other! Seeing as we only have the one camera, this was worrying as it kinda defeated the point of having the camera up close. Fortunately, sometime Friday evening I had an epiphany. I realized that I had a [Logitech C920 HD Pro Webcam](https://www.logitech.com/en-us/product/hd-pro-webcam-c920) that I use daily for video conferencing. I figured it might work in this scenario too so I fired up OBS and in no time flat had both audio and video from it setup.
+
```
Comment (jcoconnor) [inline _drafts/church-and-covid-19.md]: Nice

File: _drafts/church-and-covid-19.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+## Sunday Morning
+
+We all got to the church about an hour ahead of time. As soon as everyone was there I did a quick test recording using OBS and the Logitech C920 to check synchronization and video quality. To my relief, everything looked good... on my end, at least.
+
+### Everyone’s Streaming
+
+On Sunday March 22nd there were so many more churches streaming than usual, and so many more people watching the streams, that [StreamSpot](https://streamspot.com/) has some issues. We were able to stream without issue (thank goodness) but some of our congregation wasn’t able to watch live. I also recorded the service locally as a backup in case the stream broke partway through.
+
+> On a side note, StreamSpot’s CEO sent out a pretty awesome message that evening apologizing for the problems and assuring customers they were already working to scale their infrastructure so that there would not be a repeat of the day’s issues. I was thoroughly impressed with their transparency and forthrightness.
+
```
Comment (jcoconnor) [inline _drafts/church-and-covid-19.md]: Interesting on the 22nd - a lot of services went online at 10.00am in Northern Ireland (and probably UK wide) and Facebook video froze for a while - resolved about 10-15 minutes later.

File: _drafts/church-and-covid-19.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+### Post-service Debrief
+
```
Comment (jcoconnor) [inline _drafts/church-and-covid-19.md]: Good Agile there 👍

File: _drafts/church-and-covid-19.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+author: gene 
+title: Church and COVID-19
+---
+
+My church has been streaming our service for several years and has been live on a local FM station since before I became a member. I setup streaming initially and have been involved in all kinds of services. During that time I’ve run into just about every type of issues or challenge a sound or audio video tech in a midsize church can... or so I thought.
+
+## School and Church Closures
+
+When the local schools decided on Thursday evening, March 12th, to close down that meant our church would also close its office and cancel all events per our safety policy. Word went out on Friday the 13th that the closure would start Sunday at noon and that we would only have a single event that day: the 11am service. Our members we also advised to stay home if in any of the high-risk categories. Sitting at the soundboard that Sunday was weird... I’ve never seen an 11 o’clock service so empty.
+
```
Comment (jcoconnor) [inline _drafts/church-and-covid-19.md]: Yeah - I found it exactly the same on the last service before we closed - about 1/3 full.

File: _drafts/church-and-covid-19.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+### Going Mobile
+
+Some quick thinking was needed since the longer cables were no longer an option. The result was a decision to try streaming from my MacBook Pro. [Elgato Game Capture](https://www.elgato.com/en/gaming/game-capture-software) is what we use on the NUC so I figured I'd try installing it on my laptop. Bingo, that worked. I had also been toying with the idea of switching over to [Open Broadcaster Software](https://obsproject.com), better known as OBS, so I went ahead and installed it too. Wednesday evening I went up to the church to test out the software and to find something to mount the camera on. In the end I was able to more the existing mount for our GoPro to a spare mic stand. I tested both pieces of software and it seemed to work fine. Whew.
+
+All week, every news outlet was reiterating all the info about social distancing and emphasizing how important it was. With that in mind, during rehearsal we decided that the ministers should set a good example by not being crowded around each other... the problem was that this looked really strange. Time for a new plan.
+
+### The Oasis
+
```
Comment (jcoconnor) [inline _drafts/church-and-covid-19.md]: Just wondering would a photo insert of the area help to set the scene?

File: _drafts/church-and-covid-19.md (hunk 2 above)
Comment (jcoconnor) [reply _drafts/church-and-covid-19.md]: U have a C920 too as of 2 weeks ago :)

File: _drafts/church-and-covid-19.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+### Everyone’s Streaming
+
+On Sunday March 22nd there were so many more churches streaming than usual, and so many more people watching the streams, that [StreamSpot](https://streamspot.com/) has some issues. We were able to stream without issue (thank goodness) but some of our congregation wasn’t able to watch live. I also recorded the service locally as a backup in case the stream broke partway through.
+
+> On a side note, StreamSpot’s CEO sent out a pretty awesome message that evening apologizing for the problems and assuring customers they were already working to scale their infrastructure so that there would not be a repeat of the day’s issues. I was thoroughly impressed with their transparency and forthrightness.
+
+### Post-service Debrief
+
+Once the service was over we watched part of it and were quite pleased with the quality. My wife had watched from home and said having some people off camera was weird and we agreed. Since everyone was still around I did a quick test to see if having a second camera under the TV that showed the entire group would be better. We all agreed it looked pretty good. 
+
```
Comment (jcoconnor) [inline _drafts/church-and-covid-19.md]: Just an aside - David Schmitt has quite a bit of experience on OBS (think he might also be doing some development on it).

File: _drafts/church-and-covid-19.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+## Shelter in Place
+
+On Tuesday the 24th our county passed a shelter in place order that closed all businesses except for essential services and drive-through/takeout food. Under the order, all gathering all gatherings were also banned. This meant we were not allowed to even be at the church, much less do a broadcast from there. I welcomed this order as it was absolutely the right thing to do, but it also meant we had to reevaluate how to go about serving our congregation... again.
+
+## iPhones and iCloud to the Rescue
+
+On Wednesday our pastor posted a video to Facebook... let’s just say this was a momentous event. Aside from being a wonderful message, his video gave plausibility to an idea: he could record what he wanted to air ahead of time and send it to me via iMessage. Doing so would automagically create an iCloud sharing link that I could then use to pull down the video. This was a welcome development, for sure. I asked him for a sample so that we could both test the transfer process and so I could test the post-processing. Instead of recording a sample he did one better: he sent me the entire Wednesday video. I did some testing and determined we were in a good spot.
+
```
Comment (jcoconnor) [inline _drafts/church-and-covid-19.md]: We started low tech - our rector is using her iPhone on a selfie-stick taped to a microphone stand - will send you a photo of it.

File: _drafts/church-and-covid-19.md (hunk 2 above)
Comment (genebean) [reply _drafts/church-and-covid-19.md:31]: I really like this little camera

File: _drafts/church-and-covid-19.md (hunk 3 above)
Comment (genebean) [reply _drafts/church-and-covid-19.md:41]: It was kinda wild. I bet the internet graphs were wild looking right then.

File: _drafts/church-and-covid-19.md (hunk 4 above)
Comment (genebean) [reply _drafts/church-and-covid-19.md:43]: I totally agree!

File: _drafts/church-and-covid-19.md (hunk 5 above)
Comment (genebean) [reply _drafts/church-and-covid-19.md:11]: We had some special stuff happening so canceling really wasn't something we wanted to do but I am glad people stayed home

File: _drafts/church-and-covid-19.md (hunk 6 above)
Comment (genebean) [reply _drafts/church-and-covid-19.md:25]: If I had one, I think it would. Sadly, I don't.

File: _drafts/church-and-covid-19.md (hunk 7 above)
Comment (genebean) [reply _drafts/church-and-covid-19.md:45]: Very cool

**Pull request merged**